plugin/ranger.vim

```diff
@@ -123,6 +121,7 @@ if exists('g:ranger_replace_netrw') && g:ranger_replace_netrw
     autocmd VimEnter * silent! autocmd! FileExplorer
     autocmd StdinReadPre * let s:std_in=1
     autocmd VimEnter * if argc() == 1 && isdirectory(argv()[0]) && !exists("s:std_in") | call OpenRangerOnVimLoadDir(argv()[0]) | endif
```

Comment:
since you set the `autocmd BufEnter`, you can remove the line that set `autocmd VimEnter`. I tried with neovim and it worked for me.

Reply:
Yes, I didn't think about it, but it makes sense.